### ISSUE TITLE
[BUG] KerasSequenceValidater() does not print out val_acc results 

### ISSUE THREAD

**Issue by rnyak** [NVIDIA/NVTabular#509]
**Describe the bug**
The bug is related to `validation_callback = KerasSequenceValidater()` in the example notebooks.  Although we do pass callbacks in the tf.model.fit method as follows:

```
tf_model.fit(train_dataset_tf,  callbacks=[validation_callback], epochs=EPOCHS)
```
we do not have the validation accuracy results printed out during model training. 

I also observed that this behavior could be related to dataset size. With small sets, I do not see such behavior.

**Steps/Code to reproduce bug**
Run the rossmann and outbrain example notebooks in new_api or main branch to reproduce the bug:

**Expected behavior**
We expect the results as follow:

```
Epoch 1/10
456/456 [==============================] - 166s 364ms/step - loss: 0.4984 - binary_accuracy: 0.8067 - auc: 0.7102 - val_loss: 0.4451 - val_binary_accuracy: 0.7104 - val_auc: 0.6781
```
But we get something like:

```
Epoch 1/10
456/456 [==============================] - 166s 364ms/step - loss: 0.4984 - binary_accuracy: 0.8067 - auc: 0.7102 

```
**Environment details (please complete the following information):**
 - Environment location: NGC V100-32 GB instance using rapids conda env
 - Method of NVTabular install: Docker image nvidia/nvtabular:0.3

**Comment by rnyak** [comment]
@bschifferer for viz.

**Comment by rnyak** [comment]
@jperez999 the good example for getting eval results printed out could be Rossmann (I could get the results on a 16 GB V100 WS). 

The problematic example would be Outbrain nb.

**Comment by jperez999** [comment]
@rnyak I took a look at this and it seems we are missing a key argument when running the model fit. Please rerun you examples with the following change:
```
tf_model.fit(train_dataset_tf,  callbacks=[validation_callback], epochs=EPOCHS, validation_data=valid_dataset_tf)
```
Quick Note: to get the metrics for validation you dont even need the validation_callback. Calls into question its use_case? @benfred

**Comment by rnyak** [comment]
@jperez999 @benfred 

Alec pointed me out to this docs:

```
 Note that validation_data does not support all the data types that are supported in x, eg, dict, generator or keras.utils.Sequence.
```
https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit

Alec also wrote some explanations in this notebook : https://github.com/NVIDIA/NVTabular/blob/main/examples/tensorflow/accelerating-tensorflow.ipynb

```
The last thing I'll mention is that we're just going to do training below. The validation picture gets slightly complicated by the fact that model.fit doesn't accept Keras Sequence objects as validation data. To support this, we've built an extremely lightweight Keras callback to handle validation, KerasSequenceValidater. To see how to use it, consult the Rossmann Store Sales example notebook in the directory above this, and consider extending its functionality to support more exotic validation metrics.
```

**Comment by benfred** [comment]
Done with #701
